### ISSUE TITLE
[CI] Add Dockerfile lint testing to Travis

### ISSUE THREAD

**Issue by leifmadsen** [redhat-nfvpe/ansible-cira#51]
We should add Dockerfile lint testing now that we're starting to add some basic Docker support to the project.

The Project Atomic team has a Dockerfile linter that we can consume:

```
docker run -it --rm --privileged -v `pwd`:/root/ projectatomic/dockerfile-lint dockerfile_lint -f centos7_base
```

**Comment by dougbtv** [comment]
Alright, I have a PR incoming for this one, love this idea.

My build is failing due to the `The command "ansible-lint --exclude=roles site.yml" exited with 2.` as discussed earlier today, but, the new checks are passing. The results are available [here in my travis](https://travis-ci.org/dougbtv/ansible-cira/builds/176092109)

I made a couple small changes to the Dockerfile.
1. To pass a check for the a couple pieces of meta data, especially "[recommended labels](http://docs.projectatomic.io/container-best-practices/#_recommended_labels_for_your_project)" while not completely necessary now, they're bits of info used by k8s / openshift, and might be useful for the future. I put the "Version" as the value `{placeholder}` as we don't have any tagged releases right now. These do not impact the functionality of the container.  This check is required to be passed by the lint tool.
2. Added `yum -y clean all` as suggested by the lint.

Additionally I added the `--permissive` flag to the `dockerfile_lint` command as otherwise it fails on warnings, and I'm not sure if some of the warnings are things we want.

One of which is that it warns about having an sshd server, and for a good reason, it used to be a crutch to install a sshd server instead of using nsenter, and... now we just use `docker exec -it container_name /bin/bash` to debug a container. Unsure if we want sshd for another reason. (Running an sshd in a container is not invalid, so, there are reasonable reasons to do so)
